.github/workflows/software-versions.yaml

```diff
@@ -0,0 +1,24 @@
+---
+name: Validation of software versions referenced in ImageStream manifests
+on:  # yamllint disable-line rule:truthy
+  push:
```

Comment:
@jiridanek Since this takes so much time now, maybe I should restrict the `push` rule to the same set paths as we have for the `pull_request`? WDYT?

Reply:
maybe

possibly give it also `on: release`? https://docs.github.com/en/actions/writing-workflows/choosing-when-your-workflow-runs/events-that-trigger-workflows#release